### PR TITLE
Make getCommitMessage testable

### PR DESCRIPTION
**Pull request by chaspy**
ref: https://github.com/kamontia/qs/issues/115

### CODE REVIEW

**Comment by chaspy** [comment]
@kamontia 
Please review and merge

File: model/executer.go [hunk 1]
```diff
@@ -11,15 +13,14 @@ func (g GitCommander) Reflog(opt string) ([]byte, error) {
 }
 
 // Execute git log opts
-func (g GitCommander) Commitlog(opt string) ([]byte, error) {
-	return exec.Command("git", "log", opt).Output()
+func (g GitCommander) Commitlog(opts ...string) ([]byte, error) {
+	return exec.Command("git", "log", opts[0], opts[1]).Output()
```
Comment (kamontia) [inline model/executer.go]: It is clear that golang crashes when a part of one argument contains with space.
At this time, I merge into master, but I want to improve in the future.
Could you create the issue?

File: model/data_test.go [hunk 2]
```diff
@@ -22,3 +24,33 @@ func TestAddReflogHash(t *testing.T) {
...
+
+	// g.CommitMsgList = append(g.CommitMsgList, v)
+	got := len(gci.CommitMsgList)
+	want := 6
+	if got != want {
+		t.Fatalf("want %v, but %v:", want, got)
+	}
+
+	// g.CommitNewMsgList = append(g.CommitNewMsgList, fmt.Sprintf("fixup! %s", v))
+	gots := strings.Join(gci.CommitNewMsgList, " ")
```
Comment (kamontia) [inline model/data_test.go]: Why do you use `gots` value?
`rwant` is also.

Comment (kamontia) [reply model/data_test.go]: Please add comments details to explain the what this code tests.

Comment (chaspy) [reply model/data_test.go]: addressed and force pushed

File: model/executer.go (hunk 1 above)
Comment (chaspy) [reply model/executer.go]: 🤔

Comment (chaspy) [reply model/executer.go]: ✋ https://github.com/kamontia/qs/issues/131

**Pull request merged**